ui/raidboss/data/03-hw/trial/sophia-ex.ts

```diff
@@ -240,7 +240,7 @@ const triggerSet: TriggerSet<Data> = {
       netRegexCn: NetRegexes.startsUsing({ id: '1988', source: '信徒其一' }),
       netRegexKo: NetRegexes.startsUsing({ id: '1988', source: '제1신도' }),
       infoText: (data, matches, output) => {
-        if (Conditions.targetIsYou())
+        if (Conditions.targetIsYou()(data, matches))
```

Comment:
Sometimes I wonder if `Conditions` were a mistake.  I think this is more readable than the double parens, but feel free to disagree:

```suggestion
        if (data.me === matches.target)
```

Reply:
It's worth noting that this was written shortly after `Conditions` were introduced and before we had really solidified what we wanted out of them. Given that the usage here is inside a popup text element, rather than as a `Condition` property, I agree that the suggestion is the way we should take this.